### PR TITLE
Warehouse

### PR DESCRIPTION
**Pull request by zhangxiaotian**
<br>
--- Add a notebook to community folder to demo how to use warehouse SDK to analyze videos, index and search ---
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ Y] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ Y] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

### CODE REVIEW

File: notebooks/community/vision_ai/video_warehouse_sdk.ipynb [hunk 1]
```diff
@@ -0,0 +1,820 @@
+{
```
Comment (andrewferlitsch) [inline notebooks/community/vision_ai/video_warehouse_sdk.ipynb:1]: <p>For subfolder naming, change vision_ai to vision</p><p> </p><p>Add workbench link</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (andrewferlitsch) [inline notebooks/community/vision_ai/video_warehouse_sdk.ipynb:1]: <p>Add link (or placeholder) back to web docs.</p><p> </p><p>Learn more about [...](...)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (andrewferlitsch) [inline notebooks/community/vision_ai/video_warehouse_sdk.ipynb:1]: <p>Is there a name for the dataset. What type of dataset is it (classification, etc). What does it predict.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (andrewferlitsch) [inline notebooks/community/vision_ai/video_warehouse_sdk.ipynb:1]: <p>remove this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (andrewferlitsch) [inline notebooks/community/vision_ai/video_warehouse_sdk.ipynb:1]: <p>PROJECT_ID = "" -&gt; PROJECT_ID = "[your-project-id]"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (andrewferlitsch) [inline notebooks/community/vision_ai/video_warehouse_sdk.ipynb:1]: <p>Add the template authentication before this cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (andrewferlitsch) [inline notebooks/community/vision_ai/video_warehouse_sdk.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    PROJECT_NUMBER = int(PROJECT_NUMBER_STR[0])
</pre><p>use the template version</p><p> </p><p>change LOCATON_ID to REGION an use the template version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (andrewferlitsch) [inline notebooks/community/vision_ai/video_warehouse_sdk.ipynb:1]: <p>remove this authentication step</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (andrewferlitsch) [inline notebooks/community/vision_ai/video_warehouse_sdk.ipynb:1]: <p>make these a single cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (andrewferlitsch) [inline notebooks/community/vision_ai/video_warehouse_sdk.ipynb:1]: <p>Client -&gt; client</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Comment (andrewferlitsch) [inline notebooks/community/vision_ai/video_warehouse_sdk.ipynb:1]: <p>Use Existing -&gt; use existing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Comment (andrewferlitsch) [inline notebooks/community/vision_ai/video_warehouse_sdk.ipynb:1]: <p>a -&gt; an</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (zhangxiaotian) [reply notebooks/community/vision_ai/video_warehouse_sdk.ipynb]: Done

Comment (zhangxiaotian) [reply notebooks/community/vision_ai/video_warehouse_sdk.ipynb]: Done

Comment (zhangxiaotian) [reply notebooks/community/vision_ai/video_warehouse_sdk.ipynb]: Done

Comment (zhangxiaotian) [reply notebooks/community/vision_ai/video_warehouse_sdk.ipynb]: Done

Comment (zhangxiaotian) [reply notebooks/community/vision_ai/video_warehouse_sdk.ipynb]: I added more comments about how we use this dataset, basically our product is to demo that we can run some processing (speech transcript, text detection, embedding ) and build index and search.

Comment (zhangxiaotian) [reply notebooks/community/vision_ai/video_warehouse_sdk.ipynb]: There isn't a name for the dataset.

Comment (zhangxiaotian) [reply notebooks/community/vision_ai/video_warehouse_sdk.ipynb]: <p>hi Andrew, I updated to REGION.</p><p> </p><p>However that PROJECT_NUMBER is different from PROJECT_ID which  I am not sure what you mean by "use the template version". Thanks!</p>

Comment (zhangxiaotian) [reply notebooks/community/vision_ai/video_warehouse_sdk.ipynb]: <p>Hi Andrew,</p><p>About authentication, the reason that I used these gcloud commands is that previously we found a bug b/290266175 (it is not always reproducible) </p><p> </p><p>At that time, I was using </p><p>from google.colab import auth</p><p>auth.authenticate_user(project_id=PROJECT_ID)</p><p> </p><p>Note project_id=PROJECT_ID this part is important for us to use video intelligence api.</p><p> </p><p>But sometimes it fails as reported in b/290266175</p><p> </p><p>So I changed to use </p><p>!gcloud auth login</p><p>!gcloud auth application-default login</p><p> </p><p> </p><p> </p><p> </p>

Comment (zhangxiaotian) [reply notebooks/community/vision_ai/video_warehouse_sdk.ipynb]: marked as resolved to discuss this in another comment.

Comment (zhangxiaotian) [reply notebooks/community/vision_ai/video_warehouse_sdk.ipynb]: Mark this resolved to discuss this in the other comment.

Comment (zhangxiaotian) [reply notebooks/community/vision_ai/video_warehouse_sdk.ipynb]: changed folder and add a link. I'll go ahead test with workbench as well. mark this resolved.

Comment (zhangxiaotian) [reply notebooks/community/vision_ai/video_warehouse_sdk.ipynb]: As offline discussed, keep the project number here as it is, thanks!

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**